File: TotalAdmin/TotalAdmin.Model/DTO/EmployeeDetailDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TotalAdmin.Model
{
    public record EmployeeDetailDTO(int EmployeeNumber, string? FirstName, char? MiddleInitial, string? LastName, string? StreetAddress, string? City, string? PostalCode, string? WorkPhone, string? CellPhone, string? Email, string? JobTitle);
}

File: AngularFrontEnd/totalAdmin-app/src/app/models/employee-detail-dto.ts
export interface EmployeeDetailDTO {
    employeeNumber: number;
    firstName: string | null;
    middleInitial: string | null;
    lastName: string | null;
    streetAddress: string | null;
    city: string | null;
    postalCode: string | null;
    workPhone: string | null;
    cellPhone: string | null;
    email: string | null;
    jobTitle: string | null;
    showDetails: boolean;
}